test/features/odi-certificates.feature

```diff
@@ -0,0 +1,30 @@
+@smoke
+Feature: ODI Certificates
+
+    Scenario: DQL-67 Add Salsa organisation
```

Comment:
please remove `DQL-*` prefix and references to `Salsa`. Please use neutral org name such as `Test organisation`

Reply:
Jira story references removed and scenario outlines elaborated.
Salsa references removed and replaced with neutral org name.